src/imgui/renderer/imgui_core.cpp

```diff
@@ -52,13 +52,15 @@ void Initialize(const ::Vulkan::Instance& instance, const Frontend::WindowSDL& w
     io.ConfigFlags |= ImGuiConfigFlags_DockingEnable;
     io.DisplaySize = ImVec2((float)window.getWidth(), (float)window.getHeight());
 
-    using native_char = std::filesystem::path::value_type;
-    io.IniFilename = new char[config_path.native().size() * sizeof(native_char)];
-    io.LogFilename = new char[log_path.native().size() * sizeof(native_char)];
-    std::memcpy((void*)io.IniFilename, config_path.native().c_str(),
-                config_path.native().size() * sizeof(native_char));
-    std::memcpy((void*)io.LogFilename, log_path.native().c_str(),
-                log_path.native().size() * sizeof(native_char));
+    auto path = config_path.u8string();
+    char* config_file_buf = new char[path.size() + 1]();
+    std::memcpy(config_file_buf, path.c_str(), path.size());
```

Comment:
Still need to add the null terminator. Prob just use `path.size() + 1`

Reply:
Won't new char[...](); initialize the array to zeroes?